### PR TITLE
Make bloom size configurable for AAE fullsync.

### PR DESCRIPTION
**Pull request by jonmeredith**
Default is 1mil objects with 1% error rate.
For customers with large object counts per vnode (in the billions)
this leads to a high false positive rate causing unnecessary
transmission during the sync.

### CODE REVIEW

File: src/riak_repl_aae_source.erl [hunk 1]
```diff
@@ -223,8 +223,14 @@ prepare_exchange(start_exchange, State=#state{index=Partition}) ->
...
 update_trees(init, State) ->
-    NumKeys = 10000000,
-    {ok, Bloom} = ebloom:new(NumKeys, 0.01, random:uniform(1000)),
+    %% Temporary variables for 1.4 series to override default bloom parameters
+    %% 2.0 series will make estimates of bloom size from AAE trees.
+    %% Set aae_bloom_num_keys to estimate of number of objects in the
+    %% vnode, sum of (nval * objects with nval) forall active nval, divided
+    %% by number of vnodes.  Better to guess high.
+    NumKeys = app_helper:get_env(riak_repl, aae_bloom_num_keys, 10000000),
+    ErrorRate = app_helper:get_env(riak_repl, aae_bloom_rate, 0.1),
```
Comment (engelsanchez) [inline src/riak_repl_aae_source.erl]: Did you mean 0.01 in the default here?

Comment (jonmeredith) [inline src/riak_repl_aae_source.erl]: This is why we review - typo.  Thank you.  Small changes are the worst.

Comment (engelsanchez) [inline src/riak_repl_aae_source.erl]: what's a 0.09 difference among friends?

**Comment by engelsanchez** [comment]
Code looks good now, all eunit tests still pass. AAE fullsync riak_test passes. What else do you want? Shipz it, I says.

+1

How about fast forwarding my branch to this now?

**Comment by jonmeredith** [comment]
Merged into your branch.

**Comment by jonmeredith** [comment]
I know. Getting the 1.4 back port has been high priority for another customer. This is a workaround until customers get to a version with the estimation code.

**Pull request merged**